contracts/token/Freezable.sol

```diff
@@ -1,54 +1,38 @@
 pragma solidity ^0.5.0;
 
 /**
- * @title Freezable contract
- * @dev Base contract providing internal methods for account
- * and to track what accounts are frozen, and if whole token is frozen.
- * Token freezing is shortcut for freezing every account.
+ * @title Freezable account
```

Comment:
"@title Freezable account" should be "@title Freezable contract"

Reply:
@title Freezable account contract maybe?